python/conv/conv_1d_bench.py

```diff
@@ -18,20 +18,22 @@
 ################################################################################
 
 all_experts = [
-    SingleTilingExpert(
-        fun_name=fun_name,
-        op_name=op_name,
-        #           N  W   C  KW  F
-        tile_sizes=[1, 8, 32, 1, 8],
-        tile_interchange=[],
-        peel=[],
-        pad=False,
-        pack_paddings=[],
-        hoist_paddings=[],
-        # kwargs passed down to LowerVectors.
-        # TODO: better composition of experts.
-        transpose_lowering='shuffle',
-        print_ir_after_all=False)
+    e.print_ir(after_all=False) for e in [
+        TileAndDecompose(
+            fun_name,
+            op_name,
+            #           N  W   C  KW  F
+            tile_sizes=[1, 8, 32, 1, 8],
+            tile_interchange=[],
```

Comment:
Can we drop the unused/default entries?
We should be able to write this more concisely as:
```
TileAndDecompose(fun_name, op_name, tile_sizes=[1, 8, 32, 1, 8]) ... the rest.
```

Not for this PR but maybe as a followup when everything uses your nice fluent API as it gets close to the finish line.